packages/macros/src/with_components.rs

```diff
@@ -0,0 +1,857 @@
+            has_initializer: true,
+            has_immutable_config: false,
+            internal_impls: vec!["InternalImpl".to_string()],
+        }),
+        "Ownable" => Some(ComponentInfo {
+            name: "OwnableComponent".to_string(),
+            path: "openzeppelin_access::ownable::OwnableComponent".to_string(),
+            storage: "ownable".to_string(),
+            event: "OwnableEvent".to_string(),
+            has_initializer: true,
+            has_immutable_config: false,
+            internal_impls: vec!["InternalImpl".to_string()],
+        }),
+        "AccessControl" => Some(ComponentInfo {
+            name: "AccessControlComponent".to_string(),
+            path: "openzeppelin_access::accesscontrol::AccessControlComponent".to_string(),
+            storage: "accesscontrol".to_string(),
+            event: "AccessControlEvent".to_string(),
+            has_initializer: true,
+            has_immutable_config: false,
+            internal_impls: vec!["InternalImpl".to_string()],
+        }),
+        "Vesting" => Some(ComponentInfo {
+            name: "VestingComponent".to_string(),
+            path: "openzeppelin_finance::vesting::VestingComponent".to_string(),
+            storage: "vesting".to_string(),
+            event: "VestingEvent".to_string(),
+            has_initializer: true,
+            has_immutable_config: false,
+            internal_impls: vec!["InternalImpl".to_string()],
+        }),
+        "SRC5" => Some(ComponentInfo {
+            name: "SRC5Component".to_string(),
+            path: "openzeppelin_introspection::src5::SRC5Component".to_string(),
+            storage: "src5".to_string(),
+            event: "SRC5Event".to_string(),
+            has_initializer: false,
+            has_immutable_config: false,
+            internal_impls: vec!["InternalImpl".to_string()],
+        }),
+        "Initializable" => Some(ComponentInfo {
+            name: "InitializableComponent".to_string(),
+            path: "openzeppelin_security::InitializableComponent".to_string(),
+            storage: "initializable".to_string(),
+            event: "InitializableEvent".to_string(),
+            has_initializer: false,
+            has_immutable_config: false,
+            internal_impls: vec!["InternalImpl".to_string()],
+        }),
+        "Pausable" => Some(ComponentInfo {
+            name: "PausableComponent".to_string(),
+            path: "openzeppelin_security::PausableComponent".to_string(),
+            storage: "pausable".to_string(),
+            event: "PausableEvent".to_string(),
+            has_initializer: false,
+            has_immutable_config: false,
+            internal_impls: vec!["InternalImpl".to_string()],
+        }),
+        "ReentrancyGuard" => Some(ComponentInfo {
+            name: "ReentrancyGuardComponent".to_string(),
+            path: "openzeppelin_security::ReentrancyGuardComponent".to_string(),
+            storage: "reentrancyguard".to_string(),
```

Comment:
```suggestion
            storage: "reentrancy_guard".to_string(),
```
We should underscore for consistency, no?